Dockerfile

```diff
@@ -3,20 +3,27 @@
 #
 FROM ubuntu:16.04
 
-RUN apt-get update
-RUN apt-get install -y curl
-RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
-RUN apt-get install -y nodejs
-RUN apt-get install -y ffmpeg
-RUN apt-get install -y youtube-dl
-
-RUN mkdir -p /home/app
 WORKDIR /home/app
-RUN mkdir -p public/temp
 
-COPY . /home/app
-RUN npm install
-RUN npm run build
+RUN apt-get update \
+    && apt-get install -y curl ffmpeg \
+    && curl -sL https://deb.nodesource.com/setup_6.x | bash - \
+    && apt-get install -y nodejs \
+    && npm install -g npm@latest \
```

Comment:
Do we need to manually install npm? Afaik this is included in the node installation we do in `Line 10`

Reply:
You are installing node 6.x which has a way old version of npm